src/styles/variables.js

```diff
@@ -36,6 +36,7 @@ export default {
     iconSizeLarge: 24,
     iconSizeXLarge: 28,
     iconSizeExtraLarge: 40,
+    iconSizeMondo: 60,
```

Comment:
What does Mondo mean? I wonder if we should name this something more related to the usecase... like `iconSizeFullPageOverlay` or something?

Reply:
How about iconSizeSuperDuperLarge?

I don't think we should tie it to the use case because it may be used in a different situation where we aren't doing a full page overlay.

Reply:
Updated to iconSizeSuperLarge

Reply:
I think the current name (`iconSizeSuperLarge`) is fine.

Reply:
Cool, I don't feel strongly but at some point it would be nice to clean up these variable names.